### ISSUE TITLE
Cannot add plugins to Forge

### ISSUE THREAD

**Issue by wsw70** [itzg/docker-minecraft-server#350]
I run the image with the following configuration:

    environment:
      - EULA=TRUE
      - OPS=<someone>
      - VERSION=1.12.2
      - TYPE=FORGE

I added the `WorldEdit` mod without problems (in the `mods` folder).

I then tried to add a few plugins (specifically `Group Manager`,   `Multiverse Core 4.0.0` and `SetSpawn 2.2.1`) into the `plugins` directory. They were not found at all (I did not find anything in the logs, even information about them having been rejected).

Sorry if the question is obvious, I run servers for my children without knowing much about Minecraft :)

**Comment by itzg** [comment]
The terms "mods" and "plugins" are definitely confusing. For Forge, it will only look in the `mods` directory, which would explain why WorldEdit loaded fine for you. FYI the `plugins` directory is used by some of the other server types, such as Bukkit.

Unfortunately that in turn means Bukkit/Spigot plugins won't work with Forge (typically); however, in the case of the plugins you mentioned, ForgeEssentials seems to have those use cases covered:

https://www.curseforge.com/minecraft/mc-mods/forge-essentials-74735

**Comment by wsw70** [comment]
Thank you very much for your answer. 
Also for your fantastic, versatile docker container, it makes the life of someone who does not know Minecraft so much, much easier. Thank you.

I ended up reading a bit about Vanillia vs. Spigot vs Bukkit vs Forge vs Paper ... and while I did not find a comprehensive answer, I think I will stay with Forge for now.

As a side note, it seems that `ForgeEssentials` are dying out (http://forgeessentials.com/2017/08/the-future-of-forge-essentials/). They seem to be fine for 1.12.2 though.
